Models/ImageClassification/SqueezeNet.swift

```diff
@@ -0,0 +1,93 @@
+// Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import TensorFlow
+
+// Original Paper:
+// SqueezeNet: AlexNet Level Accuracy with 50X Fewer Parameters
+// Forrest N. Iandola, Song Han, Matthew W. Moskewicz, Khalid Ashraf, William J. Dally and Kurt Keutzer
+// https://arxiv.org/pdf/1602.07360.pdf
+
+public struct Fire: Layer {
+
+    public var squeeze: Conv2D<Float>
+    public var expand1: Conv2D<Float>
+    public var expand3: Conv2D<Float>
+
+    public init(inputFilterCount:Int, s1x1: Int, e1x1: Int, e3x3: Int) {
```

Comment:
Add a space after every colon.
```suggestion
    public init(inputFilterCount: Int, s1x1: Int, e1x1: Int, e3x3: Int) {
```